setup.py

```diff
@@ -52,6 +52,7 @@ def read(*rnames):
         'Programming Language :: Python :: 3.9',
         'Programming Language :: Python :: 3.10',
         'Programming Language :: Python :: 3.11',
+        'Programming Language :: Python :: 3.12',
```

Comment:
This is too early, until we did a security review of the Python 3.12 features, we cannot claim to support Python 3.12.

Reply:
@icemac @d-maurer did an audit of the Python 3.12 features here: https://github.com/zopefoundation/RestrictedPython/issues/246#issuecomment-1774594376

Can we go ahead?

Reply:
yes.